src/root.jl

```diff
@@ -402,7 +394,10 @@ function auto_T_JaggT(f::ROOTFile, branch; customstructs::Dict{String, Type})
         else
             leaftype = _normalize_ftype(leaf.fType)
             _type = get(_leaftypeconstlookup, leaftype, nothing)
-            isnothing(_type) && error("Cannot interpret type.")
+            if isnothing(_type)
+                @error "Cannot interpret type."
+                return nothing
```

Comment:
this line cannot be reached?

Reply:
Dammit, totally forgot about this branch. Now I was working on some data and realised that the better browsing is still not merged 😆 

Yep, that's now removed.